localstack/testing/pytest/container.py

```diff
@@ -29,6 +30,7 @@ def __call__(
         # convenience properties
         pro: bool = False,
         publish: list[int] | None = None,
```

Comment:
good point, i think we should perhaps evolve the Container fixture in general to remove some of the localstack-specific assumptions, since they should be encapsulated in the configurators. in fact i was thinking that we could maybe, when we have time, establish a new container package, something like `localstack.containers`, much like we have our `localstack.http` framework, and start moving `Container`, `RunningContainer` etc there, to make things more re-usable across localstack.